run/DockerfileUptane

```diff
@@ -49,4 +49,8 @@ RUN git clone git://git.gnome.org/libgsystem \
      gtk-doc-tools \
      python-pip
 
-RUN mkdir /sysroot && ostree admin --sysroot=/sysroot init-fs /sysroot
+RUN mkdir /sysroot \
+  && ostree admin --sysroot=/sysroot init-fs /sysroot \
+  && ostree admin --sysroot=/sysroot os-init docker \
+  && mkdir -p /sysroot/boot/loader.0 \
+  && ln -s loader.0 /sysroot/boot/loader
```

Comment:
@taheris init-fs initializes sysroot layout, os-init adds an "OS" (i.e. a group of deployments sharing a common /var directory) inside this sysroot and the last two lines take care of /boot directory: I have no idea why init-fs ignores that, but it does, so we should do it manually.